test/readrows.ts

```diff
@@ -339,37 +338,44 @@ describe('Bigtable/ReadRows', () => {
         done();
       });
     }
-    it('with an error at a fixed position', done => {
+    it('with an error at a fixed position', function (done) {
+      if (process.platform === 'win32') {
+        this.timeout(60000); // it runs much slower on Windows!
+      }
```

Comment:
nit - could this be in a beforeeach?

Reply:
I grouped this repeated code into a `setTestTimeout` function. Unfortunately, we can't use `beforeEach` because you will notice the code uses `this` which is a reference to the test context. If we put this code into a `beforeEach` hook then `this.timeout` would set the timeout on the `beforeEach` and not on the `it`.